### ISSUE TITLE
Invalid choice error for minknow config file

### ISSUE THREAD

**Issue by harisankarsadasivan** [alexomics/fillet#3]
I have installed fillet via pip and I'm doing:
<fillet sim -b /var/lib/MinKNOW/data/hela99_zymo1_11_21_19_3/helazymo_3/20191121_1843_MN28422_FAL11165_a5769093/fast5/ -t /opt/ont/minknow/conf/user_conf>

I'm getting an error as follows:

usage: fillet sim [-h] [-b BULKFILE] -t {}
fillet sim: error: argument -t: invalid choice: '/opt/ont/minknow/conf/user_conf' (choose from )

What am I doing wrong here? Note: I have not connected a device while trying this out.

**Comment by alexomics** [comment]
Yes, fillet is currently incompatible with the latest version of MinKNOW as directories have changed! I am working on a fix currently.
